backend/src/fuesim-server.ts

```diff
@@ -71,18 +71,10 @@ export class FuesimServer {
         this.saveTickInterval
     );
 
-    constructor(
-        private readonly databaseService: DatabaseService,
-        websocketPort: number = Config.websocketPort,
-        httpPort: number = Config.httpPort
```

Comment:
@ClFeSc I removed these parameters and called them inside the websocket and http server directly from the config, I haven't seen that the constructor ever gets called with `websocketPort` and `httpPort` set (therefore always being the default, maybe cypress does it.

Reply:
Maybe the initialization is not called correctly and therefore the ports aren't set to the values in test environment?

Reply:
A look at the logs would tell you that you're incorrectly importing `Config`, you have to use relative imports.

Reply:
@ClFeSc have given not the reletavie path in the imports, auto merge is on.

Reply:
@ClFeSc have given the relative path in the imports, auto merge is on.